cmd/snap/cmd_snap_op.go

```diff
@@ -342,11 +342,13 @@ func (mx *channelMixin) asksForChannel() bool {
 type modeMixin struct {
 	DevMode  bool `long:"devmode"`
 	JailMode bool `long:"jailmode"`
+	Classic  bool `long:"classic"`
 }
 
 var modeDescs = mixinDescs{
 	"devmode":  i18n.G("Request non-enforcing security"),
 	"jailmode": i18n.G("Override a snap's request for non-enforcing security"),
+	"classic":  i18n.G("Consent to use classic confinement"),
```

Comment:
@zyga Please don't get me wrong, `consent` may well be the totally right term to use, just saying that a native speaker should double check as for me it feels slightly odd but I'm not a native speaker :)